docs/development/windows_support.md

```diff
@@ -51,7 +51,7 @@ mainline, in open source, using MSVC, etc.).
 | math-libs (blas) | [hipBLASLt](https://github.com/ROCm/hipBLASLt)                               | ✅        |                                               |
 | math-libs (blas) | [rocBLAS](https://github.com/ROCm/rocBLAS)                                   | ✅        | Running tests needs PyYAML and a dll copied   |
 | math-libs (blas) | [rocSPARSE](https://github.com/ROCm/rocSPARSE)                               | ✅        | Tests need rocblas.dll and can't find files   |
-| math-libs (blas) | [hipSPARSE](https://github.com/ROCm/hipSPARSE)                               | ❔        |                                               |
+| math-libs (blas) | [hipSPARSE](https://github.com/ROCm/hipSPARSE)                               | ✅        | Same as rocSPARSE + also needs rocsparse.dll  |
```

Comment:
I'll flip `THEROCK_ENABLE_SPARSE=ON` in docs once BLAS also works: https://github.com/ROCm/TheRock/blob/251724e598a47faca9a84b08a4aa537e4635d66a/docs/development/windows_support.md?plain=1#L180-L183

I think hipBLAS still has a few lingering issues that prevent a build of `all`, but building just the `hipSPARSE` and similar utility targets works on my system.

Then later we can flip all the options in the CI/CD... once random hiccups like FFT failing on CI but not locally are fixed: https://github.com/ROCm/TheRock/blob/251724e598a47faca9a84b08a4aa537e4635d66a/.github/workflows/build_windows_packages.yml#L126-L131